fractal_server/app/api/v1/workflowtask.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2022 (C) Friedrich Miescher Institute for Biomedical Research and
+# University of Zurich
+#
+# Original author(s):
+# Jacopo Nespolo <jacopo.nespolo@exact-lab.it>
+# Marco Franzon <marco.franzon@exact-lab.it>
+# Tommaso Comparin <tommaso.comparin@exact-lab.it>
+# Yuri Chiucconi <yuri.chiucconi@exact-lab.it>
+#
+# This file is part of Fractal and was originally developed by eXact lab S.r.l.
+# <exact-lab.it> under contract with Liberali Lab from the Friedrich Miescher
+# Institute for Biomedical Research and Pelkmans Lab from the University of
+# Zurich.
+from copy import deepcopy
+from typing import Optional
+
+from fastapi import APIRouter
+from fastapi import Depends
+from fastapi import HTTPException
+from fastapi import Response
+from fastapi import status
+
+from ...db import AsyncSession
+from ...db import get_db
+from ...models import WorkflowTaskCreate
+from ...models import WorkflowTaskRead
+from ...models import WorkflowTaskUpdate
+from ...security import current_active_user
+from ...security import User
+from ._aux_functions import _get_workflow_check_owner
+from ._aux_functions import _get_workflow_task_check_owner
+
+router = APIRouter()
+
+
+@router.post(
+    "/project/{project_id}/workflow/{workflow_id}/wftask/",
+    response_model=WorkflowTaskRead,
+    status_code=status.HTTP_201_CREATED,
+)
+async def create_workflowtask(
+    project_id: int,
+    workflow_id: int,
+    task_id: int,
+    new_task: WorkflowTaskCreate,
+    user: User = Depends(current_active_user),
+    db: AsyncSession = Depends(get_db),
+) -> Optional[WorkflowTaskRead]:
+    """
+    Add a WorkflowTask to a Workflow
+    """
+
+    workflow = await _get_workflow_check_owner(
+        project_id=project_id, workflow_id=workflow_id, user_id=user.id, db=db
+    )
+    async with db:
+        workflow_task = await workflow.insert_task(
+            **new_task.dict(),
+            task_id=task_id,
+            db=db,
+        )
+
+    await db.close()
+    return workflow_task
+
+
+@router.get(
+    "/project/{project_id}/workflow/{workflow_id}/wftask/{workflow_task_id}",
+    response_model=WorkflowTaskRead,
+)
+async def get_workflowtask(
```

Comment:
At least when we update a file, let's move to CRUD-notation function names (here: `read_workflowtask`)